.github/workflows/python.yml

```diff
@@ -0,0 +1,33 @@
+name: Python CI
+
+on:
+  schedule:
+    # each fifteen day of month
+    - cron: "0 0 15 * *"
+  push:
+    paths:
+      - "python/**"
+      - ".github/**"
+  pull_request:
+    paths:
+      - "python/**"
+      - ".github/**"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: astral-sh/setup-uv@v6
+        with:
+          version: "latest"
+
+      - name: all tests except the expected failing one
+        working-directory: python
+        run: uv run pytest -k "not test_order_cannot_be_updated_or_canceled_if_paid" -v
```

Comment:
The Java workflow calls maven and checks if the test that is expect to fail really did fail. https://github.com/sammancoaching/CoffeeShop-TestDesign-Kata/blob/93a449c1a4f5b703b12b4d976ba51bec5db569b1/.github/workflows/maven.yml#L33

Unfortunately, doing this is not really nicely possible from the code (ignoring the mvn failure and grep in the surefire output). 
Here in python only the non-failing tests are run, so we don't know if the test that should fail in the build actually failed. 
Would it be possible to do this in the python build as well? Would be nice, but is not a must.

Reply:
So that's just a CI-nice-to-have-issue and has nothing to do with the content itself.

Reply:
we do both, next step of the CI checks that the failing test is failing

Reply:
Ah, I think now I got it: `test_order_cannot_be_updated_or_canceled_if_paid` is the failing test, right? So you are executing exactly the failing test and the exclamation mark negates the `rc` of the command so the command is successful if it failed, is that what the line does?